Update route-server-limits.md (#97488)
Azure/azure-content: includes/route-server-limits.md

|----------|-------|
| Number of BGP peers supported | 8 |
| Number of routes each BGP peer can advertise to Azure Route Server <sup>1</sup> | 1000 |
| Number of routes that Azure Route Server can advertise to ExpressRoute or VPN gateway <sup>2</sup>  | 200 |
| Number of VMs in the virtual network (including peered virtual networks) that Azure Route Server can support <sup>3</sup> | 2000 |

<sup>1</sup> If your NVA advertises more routes than the limit, the BGP session will get dropped. In the event BGP session is dropped between the gateway and Azure Route Server, you'll lose connectivity from your on-premises network to Azure.

> <sup>2</sup> Please be aware of the ExpressRoute Private Peering [limit](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#:~:text=for%20an%20ExpressRoute%20connection-,1%2C000,-Maximum%20number%20of%20IPv6) of 1000 routes per connection from Virtual Network Gateway towards ExpressRoute circuit. I.e. The total number of routes from all Virtual Network Address Spaces + ARS Branch-to-branch, must not exceed 1000.

Reviewer: ```suggestion
<sup>2</sup> Please be aware of the ExpressRoute Private Peering [limit](/azure/azure-resource-manager/management/azure-subscription-service-limits#:~:text=for%20an%20ExpressRoute%20connection-,1%2C000,-Maximum%20number%20of%20IPv6) of 1000 routes per connection from Virtual Network Gateway towards ExpressRoute circuit. For instance, the total number of routes from all Virtual Network Address Spaces + ARS Branch-to-branch, must not exceed 1000.
```
